app_dart/lib/src/service/config.dart

```diff
@@ -46,7 +46,7 @@ class Config {
   /// This adds support for the `waiting for tree to go green label` to the repo.
   ///
   /// Relies on the GitHub Checks API being enabled for this repo.
-  static Set<RepositorySlug> supportedRepos = <RepositorySlug>{
+  Set<RepositorySlug> supportedRepos = <RepositorySlug>{
```

Comment:
This needed to be changed so tests could override it. Otherwise, the data would look weird where there would be 6 backfills instead of 1 (1 for each supported repo)